plugins/plugins64.xml

```diff
@@ -202,7 +202,7 @@
 		</install>
 	</plugin>
 	<plugin name="CS-Script (C# intellisense)">
-		<x64Version>1.7.11.0</x64Version>
+		<x64Version>1.7.11.1</x64Version>
```

Comment:
@oleg-shilo There is no such release. What is the intention of this?

Reply:
Hi @chcg,

You are right, the latest product release is v1.7.11.0. But unfortunately I have made a mistake in the last PR where the the package URL was miss typed. I assumed that in order to fix it I had to increment the version in one or another way. That's why I incremented that build component of the version.

But I am absolutely happy with your fix f8291a0. Please let me know if anything else needs to be done from my side.

Reply:
ok, no further action needed. This PR could be closed then.